.codesandbox/ci.json

```diff
@@ -1,11 +1,12 @@
 {
-  "buildCommand": "build:min",
+  "buildCommand": "build --to @fluentui/react --to @fluentui/react-components --to @fluentui/react-northstar",
```

Comment:
q: why do we need to build vNext for codesandbox ? AFAIR it was solely for Fabric ? hmm